tests/sls/homu/valid_repos.py

```diff
@@ -0,0 +1,33 @@
+import toml
+import urllib.request
+from urllib.error import URLError
+from tests.util import Failure, Success
+
+
+def getStatus(url):
```

Comment:
Let's make this a `repo_exists` method, which takes a repo identifier string in the form `owner/name`, e.g. `servo/saltfs`, and returns a boolean. Update the doc string as appropriate.